build/path.js

```diff
@@ -32,6 +32,9 @@ module.exports = {
     testOutputDir: path.resolve(rootPath, 'test'),
     aliases: {
         ui: srcDir,
-        build: path.resolve(rootPath, 'build')
+        build: path.resolve(rootPath, 'build'),
+        // Hardcoded paths to partial Handlebars templates (prevents unresolved template dependencies)
+        'ui-generis-widget-label.tpl': path.resolve(srcDir, 'generis/widget/widget.tpl'),
+        'ui-form-widget-label.tpl': path.resolve(srcDir, 'form/widget/tpl/label.tpl')
```

Comment:
I don't understand why those 2 templates need to have an alias ?

Reply:
Those are the only 2 partial templates in TAO CE. When they are called from another template (https://github.com/oat-sa/tao-core-ui-fe/blob/master/src/form/widget/tpl/checkBox.tpl#L3), Rollup doesn't figure out how to resolve the module. Adding aliases was the simplest way to solve that.

It's supposed to be possible using https://github.com/mixmaxhq/rollup-plugin-handlebars-plus, but I also found that method doesn't work without substantial changes (files to be renamed to match their template name, references changed, list of partial templates defined in the Rollup config etc).